docs/user-guide/admin-cli.md

```diff
@@ -317,22 +317,20 @@ Remove a tenant, optionally all volumes for a tenant can be removed as well.
 
 Sample:
 ```
-/usr/lib/vmware/vmdkops/bin/vmdkops_admin.py tenant rm tenant1 --remove-volumes True
+/usr/lib/vmware/vmdkops/bin/vmdkops_admin.py tenant rm --name=tenant1 --remove-volumes
```

Comment:
@kerneltime - `vmdkops_admin tenant T1 access add --rights all` is not trivial with argparse package. @andrewjstone - can you check if the above is true :-) ?

I suggest leaving it as is